apps/researcher/src/app/[locale]/objects/[id]/date-created-to-string.ts

```diff
@@ -0,0 +1,24 @@
+import {TimeSpan} from '@/lib/api/definitions';
+
+export default function dateCreatedToString(
```

Comment:
I deliberately chose to make this function specific for `dateCreated` because there has been some discussion about how to display dateCreated. For now, it is a pretty straightforward time span, but I think this is going to change in the future.

Reply:
That's fine! I'd still recommend changing the suffix of the function name `dateCreatedToString` though, to make the meaning of the transformation clear (from a date to a formatted date/localized date).